cryptoki/tests/basic.rs

```diff
@@ -1862,6 +1862,123 @@ fn aes_gcm_with_aad() -> TestResult {
     Ok(())
 }
 
+#[test]
+#[serial]
+fn encrypt_decrypt_gcm_message_no_aad() -> TestResult {
+    let (pkcs11, slot) = init_pins();
+    // PKCS#11 3.0 API is not supported by this token. Skip
+    if !pkcs11.is_fn_supported(Function::MessageEncryptInit) {
+        /* return Ignore(); */
```

Comment:
I think it'd be good to add a `println` here saying that the test is ignored or something (just to give a way for the dev to see what was executed or not).

Reply:
Yes, this (along others) is one of the issues I have with `cargo test`.

FWIW it seems the issue is tracked at https://github.com/rust-lang/rust/issues/68007.

Reply:
I hope I will be able to remove the other skips as soon as I will update kryptic with the recent fixes so I changed it only now in these two cases I added.